Makefile

```diff
@@ -8,18 +8,20 @@ SOURCE      := $$PWD
 EXT_ID      := paperwm@paperwm.github.com
 EXT_DIR     := $(XDG_DATA_HOME)/gnome-shell/extensions
 TARGET      := $(EXT_DIR)/$(EXT_ID)
+SYSTARGET   := $(DESTDIR)/usr/share/gnome-shell/extensions/$(EXT_ID)
```

Comment:
This and the `install`/`install-system` dichotomy are really unnecessary -- a system-wide builder will update `TARGET` to point to `/usr/share/gnome-shell/extensions/...` (see how our `default.nix` does it)

At worst, I'd add a `GSCHEMA_TARGET` and have the `install` target try to install the schemas there if nonempty. This way complexity remains at a minimum and the Makefile can easily be made to install gschemas system-wide as desired.